Doc/howto/logging.rst

```diff
@@ -26,9 +26,11 @@ When to use logging
 ^^^^^^^^^^^^^^^^^^^
 
 Logging provides a set of convenience functions for simple logging usage. These
```

Comment:
Since your change refers to using logger methods rather than the convenience functions, I'd remove this sentence and go with something like "You can access logging functionality by creating a logger via ..."

Reply:
Done